nncf/experimental/post_training/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -0,0 +1,366 @@
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPointsContainer
+
+ModelType = TypeVar('ModelType')
+
+
+class FastBiasCorrectionParameters(AlgorithmParameters):
+    """
+    Parameters of FastBiasCorrection algorithm
+
+    :param number_samples: The number of the samples for the statistics collection.
+    :param threshold: The magnitude threshold that regulates the application of the shift.
+    """
+
+    def __init__(self, number_samples: int = 100, threshold: float = 2.0) -> None:
+        """
+        Initializes FastBiasCorrectionParameters
+
+        :param number_samples: The number of the samples for the statistics collection.
+            This statistics uses for the further calculation of the bias shift.
+        :param threshold: The magnitude threshold that regulates the application of the shift.
+            Magnitude calculates as the maximum of the absolute ratio of the shift to the original bias value.
+            If the calculated value less than threshold, shift will apply to the bias.
+        """
+        self.number_samples = number_samples
+        self.threshold = threshold
+
+    def to_json(self) -> Dict[str, Union[str, float, int]]:
+        """
+        Serialize all FastBiasCorrection parameters to JSON.
+        """
+
+
+class FastBiasCorrection(Algorithm):
+
+    """
+    Post-training FastBiasCorrection algorithm implementation
+
+    The main purpose of this algorithm to reduce quantization error
+    via correction the bias of the Convolutions, FullyConnected, etc. layers.
+    The algorithm pipeline is very simple:
+        - we collects floating-point statistics from the corresponding model for the layers with bias;
+        - then we gets the quantized model and try to reduce it's error by correction of the bias;
+        - the shift calculates using the sub-graph that consists of the correction layer and
+        weight quantizer-dequantizer pair or fake quantize node;
+        - the floating-point statistics uses as input for
+        the sub-graph and further quantization output calculation;
+        - in the end we corrects the original bias by the difference (shift)
+        between floating-point and quantized outputs.
+
+    :param number_samples: The number of the samples for the statistics collection.
+    :param threshold: The magnitude threshold that regulates the application of the shift.
+    :param nncf_graph: NNCFGraph class for the algorithm.
+    """
+
+    def __init__(self, parameters: FastBiasCorrectionParameters):
+        """
+        Initializes FastBiasCorrection
+
+        :param parameters: The instance of the FastBiasCorrectionParameters.
+        """
+        super().__init__()
+        self.number_samples = parameters.number_samples
+        self.threshold = parameters.threshold
+        self.nncf_graph = None
+        self._backend_entity = None
+
+    @property
+    def available_backends(self) -> List[BackendType]:
```

Comment:
Needs a docstring. Also the actual returned value is inconsistent with the return type, at least by name.

Reply:
The description of the method in the base class is not enough?

Reply:
Ok, missed the base class docstring. Check the return type, though